pkg/gds/interceptor.go

```diff
@@ -18,10 +21,35 @@ func (s *Service) serverInterceptor(ctx context.Context, in interface{}, info *g
 	start := time.Now()
 	panicked := true
 
+	trackPanics := false
+	trackPerfomance := false
+	switch info.Server.(type) {
+	case api.TRISADirectoryServer:
+		if s.conf.API.Sentry.Enabled {
+			trackPanics = true
+			if s.conf.API.Sentry.TrackPerformance {
+				trackPerfomance = true
+			}
+		}
+	case members.TRISAMembersServer:
+		if s.conf.Members.Sentry.Enabled {
+			trackPanics = true
+			if s.conf.Members.Sentry.TrackPerformance {
+				trackPerfomance = true
+			}
+		}
+	default:
+		log.WithLevel(zerolog.PanicLevel).Err(fmt.Errorf("unknown service type: %T", info.Server))
+		return nil, status.Error(codes.Unimplemented, "unknown service type for request")
+	}
```

Comment:
I think that's reasonable, this part of the code is definitely a bit weird and since we're using the same interceptor anyway for these it seems that they should be considered together in terms of error and performance tracking.

Reply:
Actually scratch that, I think the best thing to do for now is to put the monolith (GDS API, Admin API, Members API) into its own Sentry. It simplifies the configuration a lot and it looks like it's more difficult than I originally thought to separate them. If we feel we need to separate them then we can create a separate story for that.